Gemfile

```diff
@@ -1,30 +1,28 @@
 # frozen_string_literal: true
+
 source 'https://rubygems.org'
 
-ruby '2.3.1', engine: 'jruby', engine_version: '9.1.5.0' if ENV.key?('DYNO')
+ruby '2.4.1' if ENV.key?('DYNO')
 
 def gh(slug)
   "https://github.com/#{slug}.git"
 end
 
 gem 'activesupport', '~> 3'
 gem 'aws-sdk'
-gem 'bunny', platform: :mri
+gem 'bunny'
 gem 'coder', git: gh('rkh/coder')
 gem 'concurrent-ruby', require: 'concurrent'
 gem 'connection_pool'
 gem 'excon'
 gem 'faraday'
-gem 'jdbc-postgres', platform: :jruby
-gem 'jrjackson', platform: :jruby
-gem 'jruby-openssl', platform: :jruby
-gem 'json'
+gem 'jemalloc'
```

Comment:
I have a fork of this gem at https://github.com/joshk/jemalloc-rb which uses a much newer version of jemalloc, in case you want to use it